pages/index.js

```diff
@@ -6,6 +6,7 @@ export default function Home() {
       <Head>
         <title>Summer of Shipping</title>
         <link rel="icon" href="/favicon.ico" />
+        <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet"></link>
```

Comment:
The more I look at Raleway, the more the "W" bothers me. No need to change it now, but I'm actively thinking about alternative fonts

Reply:
Changed the font to Roboto for next commit

Other options to consider are: Open Sans, Source Sans Pro, Roboto Slab, Montserrat